packages/barclay-integration/e2e/barclay.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    PaymentStepAsGuestPreset,
+    test,
+} from '@bigcommerce/checkout/payment-integration-test-framework';
+
+test.describe('Offsite payment method', () => {
+    test('Barclay is working', async ({ assertions, checkout }) => {
+        // Testing environment setup
+        await checkout.use(new PaymentStepAsGuestPreset());
+        await checkout.start('Barclay');
+        await checkout.route('**/pay/initialize', `${__dirname}/support/barclay.ejs`);
```

Comment:
No need for a mockup, but the third-party site had to be replaced with a local page.